imap/conversations.h

```diff
@@ -97,6 +97,7 @@ struct conversations_state {
     char *annotmboxname;
     strarray_t *counted_flags;
     strarray_t *folders;
+    strarray_t *altnames;  // cache the alternative mboxname or uniqueid
```

Comment:
nit: might want to rename this to altrep

Reply:
done.